packages/extension/src/newtab/DndModal.tsx

```diff
@@ -113,36 +80,36 @@ export default function DndModal({
     return `daily.dev in a new tab is paused, it will resume on ${date} at ${time}`;
   };
 
-  const titleSize = isActive ? 'typo-title3' : 'typo-title2';
-
   return (
-    <StyledModal
+    <Modal
+      kind={Modal.Kind.FlexibleTop}
+      size={Modal.Size.Medium}
       {...modalProps}
       onRequestClose={onRequestClose}
-      style={{
-        content: {
-          paddingTop: '2rem',
-          maxWidth: '27.5rem',
-          display: 'flex',
-          flexDirection: 'column',
-        },
-      }}
     >
-      {!isActive && <ModalCloseButton onClick={onRequestClose} />}
-      <div className="px-16 text-center">
-        <h3
-          className={classnames(
-            'font-bold text-theme-label-primary',
-            titleSize,
-          )}
-        >
-          Pause new tab
-        </h3>
+      <Modal.Header title="Pause new tab" />
+      <Modal.Body>
         <p className="mt-1 typo-callout text-theme-label-secondary">
           {getDescription()}
         </p>
-      </div>
-      {isActive ? turnedOnContent : turnedOffContent}
-    </StyledModal>
+        {!isActive ? turnedOffContent : null}
```

Comment:
I think we should move the turnedoffcontent here as well, or convert it into a component.

Reply:
We could simply show it here, let me refactor :)